bedevere/bpo.py

```diff
@@ -49,7 +44,12 @@
                 new_body = BODY.format(body=body, issue_number=issue_number)
                 body_data = {"body": new_body, "maintainer_can_modify": True}
                 await gh.patch(event.data["pull_request"]["url"], data=body_data)
-        status = create_success_status(issue_number_found)
+        issue_number_on_bpo = (await
```

Comment:
I wouldn't split this line. Just assign `issue_number_found.group()` to a variable and then pass that into the `await` call. That will probably also shorten it enough to be able to put it in the first line of the `if`.

Reply:
Are you referring to lines 37-38 or 47-48?